nixos/doc/manual/release-notes/rl-2211.section.md

```diff
@@ -272,6 +272,12 @@ Available as [services.patroni](options.html#opt-services.patroni.enable).
 
 - The `services.matrix-synapse` systemd unit has been hardened.
 
+- The `services.grafana` options were converted to a [RFC 0042](https://github.com/NixOS/rfcs/blob/master/rfcs/0042-config-option.md) configuration.
```

Comment:
This changelog entry is pretty much useless. The changelog should help users updating to figure out what they need to change. This technical justification is completely irrelevant – it should much rather tell people what they can expect to change and ideally how they need to change what in order to be able to update.

You shouldn't need to read an RFC just to get a vague idea what you need to change in your grafana configuration.